### PR TITLE
feat(route): add route of Oct的小破站

### LINKED ISSUE #16804: 求大佬给我的博客加一下RSS👉Oct0pu5.cn
### 类型

博客

### 网站地址

https://oct0pu5.cn/

### 网站描述

我的个人博客

### 需要生成什么内容？

标题，时间，全文（保留格式）

### 额外描述

我自己写了一点代码，不知道对不对，烦请大佬指正，球球了orz

代码在我的仓库，lib/routes/oct0pu5，非常感谢！😘

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by wiketool**
## Involved Issue / 该 PR 相关 Issue

Close #16804

## Example for the Proposed Route(s) / 路由地址示例

```routes
/oct0pu5
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [ ] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

**Comment by Konano** [comment]
RSSHub 能添加个人博客的 route 吗？应该不可以吧。

File: lib/routes/oct0pu5/rss.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        params: {
+            title: '博客',
+            description: 'Oct0pu5的博客',
+        },
+        item: {
+            item: '.recent-posts > .recent-post-item',
+            title: `$('.recent-post-info > a').text()`,
+            link: `$('.recent-post-info > a').attr('href')`,
+            description: `$('.recent-post-info > .content').text()`,
+            pubDate: `Date.parse($('div.recent-post-info > div.article-meta-wrap > span.post-meta-date > time').text().trim())`,
```
Comment (TonyRL) [inline lib/routes/oct0pu5/rss.ts:45]: Please use `parseDate` instead.
docs: https://docs.rsshub.app/joinus/advanced/pub-date

Comment (TonyRL) [reply lib/routes/oct0pu5/rss.ts:45]: fixed in https://github.com/DIYgod/RSSHub/commit/02efe692b2011a3ad22e6d2ac7a434f4f63fc5c7

Comment (Octopus058) [reply lib/routes/oct0pu5/rss.ts:45]: 感谢dalao

Comment (wiketool) [reply lib/routes/oct0pu5/rss.ts]: > fixed in [02efe69](https://github.com/DIYgod/RSSHub/commit/02efe692b2011a3ad22e6d2ac7a434f4f63fc5c7)

It seems still not working https://github.com/DIYgod/RSSHub/pull/16809#issuecomment-2362589033 even after i rebase your commit https://github.com/DIYgod/RSSHub/commit/02efe692b2011a3ad22e6d2ac7a434f4f63fc5c7

Comment (Octopus058) [reply lib/routes/oct0pu5/rss.ts]: > > fixed in [02efe69](https://github.com/DIYgod/RSSHub/commit/02efe692b2011a3ad22e6d2ac7a434f4f63fc5c7)
> 
> It seems still not working [#16809 (comment)](https://github.com/DIYgod/RSSHub/pull/16809#issuecomment-2362589033) even after i rebase your commit [02efe69](https://github.com/DIYgod/RSSHub/commit/02efe692b2011a3ad22e6d2ac7a434f4f63fc5c7)

maybe should add this?
```python
import { parseDate } from '@/utils/parse-date';
```

**Pull request merged**